### PR TITLE
Add drag to RichTextLabel

### PR DESCRIPTION
**Pull request by ConteZero**
I've implemented the drag operation on RichTextLabel so it is possible to copy selected text into other nodes that support drag and drop.

### CODE REVIEW

File: scene/gui/rich_text_label.cpp [hunk 1]
```diff
@@ -1501,6 +1502,9 @@ void RichTextLabel::_notification(int p_what) {
 				update();
 			}
 		} break;
+		case Control::NOTIFICATION_DRAG_END: {
```
Comment (KoBeWi) [inline scene/gui/rich_text_label.cpp:1505]:
```suggestion
		case NOTIFICATION_DRAG_END: {
```
`NOTIFICATION_DRAG_END` is defined in Node, the `Control::` part isn't necessary.
(your previous PRs seem to have done the same)

**Review by KoBeWi — APPROVED** [review]
Works fine. Code looks ok too, but needs a review from someone who knows RichtTextLabel better.

**Review by bruvzg — APPROVED** [review]
Seems to be working fine, I have added one suggestion, but it should be OK without it as well.

File: scene/gui/rich_text_label.cpp (hunk 1 above)
Comment (ConteZero) [reply scene/gui/rich_text_label.cpp]: done

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**